aladdin-container.sh

```diff
@@ -118,8 +98,13 @@ function environment_init() {
 }
 
 function _is_cluster_ready() {
-    # Cluster is ready if we are on LOCAL or if we can pull kube config via kops
-    "$IS_LOCAL" || kops export kubecfg --name $CLUSTER_NAME --admin
+    # If not local, check if we can pull kube config via kops to determine if remote cluster is ready
+    if ! "$IS_LOCAL"; then
+        # Allow to use a different aws profile from aladdin config for kops command
```

Comment:
Kops had an issue with its aws sdk being weird with sso, so it needed the option for a dedicated aws profile